single-node-k8s.sh

```diff
@@ -70,6 +70,8 @@ function set_env() {
     # downloading rook images is sometimes slow, extend timeout to 15 minutes
     export ROOK_VERSION='v1.3.9'
     export ROOK_DEPLOY_TIMEOUT=900
+    # use podman for minikube.sh, Docker is not installed on the host
+    export CONTAINER_CMD='podman'
```

Comment:
@nixpanic is it a good idea to replace the hardcoded use of `podman` in this file with CONTAINER_CMD variable?

Reply:
No, we do not really need to have it configurable. And ideally we never use `docker` anywhere.